src/transformers/models/dpr/modeling_tf_dpr.py

```diff
@@ -596,6 +549,7 @@ def get_input_embeddings(self):
 
     @add_start_docstrings_to_model_forward(TF_DPR_ENCODERS_INPUTS_DOCSTRING)
     @replace_return_docstrings(output_type=TFDPRContextEncoderOutput, config_class=_CONFIG_FOR_DOC)
+    @unpack_inputs
```

Comment:
Sorry to nag you about this -- the other PRs have placed the new decorator as the top decorator. For consistency, can you move it so as to be in line with the other models? 🙏

Reply:
No Issue! Let me change and push it!